### ISSUE TITLE
Octopus Deploy library Step Template

### ISSUE THREAD

**Issue by jzabroski** [fluentmigrator/fluentmigrator#1034]
Some competitors have official versions on the Octopus Deploy app store:
- [Flyway Migrate](https://library.octopus.com/step-templates/3868c17c-8471-4805-89be-e99513afd59a/actiontemplate-flyway-migrate)
- [Run EntityFramework Migrations (Update-Database)](https://library.octopus.com/step-templates/a6cd35d6-164a-4e57-a0f8-0d327129783a/actiontemplate-run-entity-framework-migrations-(update-database))
- Various RedGate and ApexSQL plug-ins, such as ReadyToRoll

**Comment by fubar-coder** [comment]
I suggest to increase the reliability of the runners first.

**Comment by jzabroski** [comment]
I agree - this is not something to work on at the moment, just for tracking purposes.
